src/components/CarouselSection/CarouselSection.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import NewsCarousel from "./NewsCarousel/NewsCarousel";
+import { CMSContext } from "@/contexts/ContentProvider";
+import {
+  CarouselWrapper,
+  CarouselWrapperSection,
+} from "./CarouselSection.styles";
+import { SectionTitle } from "@/app/globalStyles";
+import { useContext, useEffect, useState } from "react";
+import { INews } from "@/utils/interfaces";
+
+const CarouselSection = () => {
+  const { loadData } = useContext(CMSContext);
+  const [newsItems, setNewsItems] = useState<INews[]>([]);
+  const [loading, setLoading] = useState(true);
+
+  const loadNews = async () => {
+    const fetchedNewsItems = await loadData("news");
+    if (fetchedNewsItems.length > 0) {
+      setNewsItems(fetchedNewsItems as unknown as INews[]);
+    } else {
+      setNewsItems([
```

Comment:
I think it's best if we added this "default" value as the initial value of the newItems state, with it we won't need the "else" statement